doc/docs/learn/create-an-app-with-monomer.md

```diff
@@ -0,0 +1,568 @@
+---
+sidebar_position: 2
+---
+
+# Create an Application With Monomer
+
+In this tutorial, you will learn how to create a new Monomer application with
+a Cosmos SDK [simapp](https://docs.cosmos.network/main/learn/intro/sdk-design#baseapp) (or, `baseapp`).
+
+## Setup
+
+Monomer is a framework for deploying Cosmos SDK applications as rollups on
+Ethereum. As such, the development environment for Monomer is similar to that of
+a Cosmos SDK application. To begin building an application with Monomer, we can
+start with a basic Cosmos SDK app. 
+
+We'll use the handy tool [spawn](https://github.com/rollchains/spawn) to bootstrap a new Cosmos SDK application. If you do not already have it, follow the instructions below to install it.
+
+<details>
+  <summary>Install `spawn`</summary>
+
+```bash
+git clone https://github.com/rollchains/spawn
+cd spawn
+git checkout v0.50.4
+make install
+```
+</details>
+
+
+### Bootstrapping a Cosmos SDK Application
+We can spawn a new Cosmos SDK application with `spawn new`:
+```bash
+spawn new rollchain --consensus=proof-of-authority \
+    --bech32=roll \
+    --denom=uroll \
+    --bin=rolld \
+    --disabled=cosmwasm,globalfee \
+    --org=monomer
+```
+
+This gives us a boilerplate Cosmos SDK application in the `rollchain`
+directory.
+
+## Integrating Monomer
+Let's move into the `rollchain` directory and add Monomer to our application.
+
+```bash
+cd rollchain
+go get github.com/polymerdao/monomer
```

Comment:
Do we need a `go mod tidy` after this? I got some errors when pasting the imports.

Reply:
Hmm we shouldn't but it might be good to do so. Let me run through this again with a fresh `rollchain` and see if I run into those same errors.